gradle.properties

```diff
@@ -6,3 +6,5 @@ errorproneVersion=2.3.4
 errorproneJavacVersion=9+181-r4173-1
 
 android.enableUnitTestBinaryResources=true
+
+org.gradle.vfs.watch=true
```

Comment:
What effect does this have? Is there an alternative way to enable this via a command-line option?

Reply:
This enables a new feature in the gradle daemon where it'll monitor the filesystem for changes on an ongoing basis rather than re-evaluating the state of the filesystem on each build. It's beneficial during development because it speeds up incremental builds, but won't show much benefit where all builds are from a cold state (e.g. CI). 

There is a command line switch (--watch-fs), and if you'd prefer I can pull this change from the PR.